taxonomies/categories.txt

```diff
@@ -5582,7 +5582,7 @@ nl:Gearomatiseerde appelsappen
 
 <en:Apple juices
 <en:Squeezed juices
-en:Squeezed apple juices, Pressed apple juices, Non-refrigerated squeezed apple juices, Non-refrigerated pressed apple juices, Pure apple juice
+en:Pure apple juices, Squeezed apple juices, Pressed apple juices, Non-refrigerated squeezed apple juices, Non-refrigerated pressed apple juices, Pure apple juice
```

Comment:
@aleene Just to understand, this is for all apple juices that are not reconstituted from concentrate?

Reply:
Yes